<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LevelView</class>
 <widget class="QMainWindow" name="LevelView">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>580</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Untitled</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="2">
     <widget class="QLabel" name="label">
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../Resource/LFEditor.qrc">:/MainView/Level.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="TitleLabel">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Level</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QScrollArea" name="TilesScroll">
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <widget class="QWidget" name="TilesVerticalLayout">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>138</width>
            <height>318</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="sizeConstraint">
            <enum>QLayout::SetFixedSize</enum>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QScrollArea" name="LevelMatrixScroll">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <widget class="QWidget" name="LevelMatrixGridLayout">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>396</width>
            <height>318</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_2" columnminimumwidth="0">
           <property name="sizeConstraint">
            <enum>QLayout::SetFixedSize</enum>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QWidget" name="widget_2" native="true">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Level index</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="LevelIndexSpin">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>99</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Zoom</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="ZoomSlider">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>10</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksBelow</enum>
            </property>
            <property name="tickInterval">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="ActionNew"/>
   <addaction name="ActionOpen"/>
   <addaction name="ActionSave"/>
  </widget>
  <action name="ActionNew">
   <property name="icon">
    <iconset resource="../Resource/LFEditor.qrc">
     <normaloff>:/MainView/new.png</normaloff>:/MainView/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="ActionOpen">
   <property name="icon">
    <iconset resource="../Resource/LFEditor.qrc">
     <normaloff>:/MainView/open.png</normaloff>:/MainView/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="ActionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../Resource/LFEditor.qrc">
     <normaloff>:/MainView/save.png</normaloff>:/MainView/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../Resource/LFEditor.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>toolBar</sender>
   <signal>actionTriggered(QAction*)</signal>
   <receiver>LevelView</receiver>
   <slot>ToolButtonClick(QAction*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>376</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>411</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ZoomSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>LevelView</receiver>
   <slot>ZoomValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>427</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>458</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LevelIndexSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>LevelView</receiver>
   <slot>LevelIndexValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>423</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>460</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ToolButtonClick(QAction*)</slot>
  <slot>TilesComboIndexChanged(QString)</slot>
  <slot>LevelIndexValueChanged(int)</slot>
  <slot>ZoomValueChanged(int)</slot>
 </slots>
</ui>
